Formula/pyroscope.rb

```diff
@@ -0,0 +1,63 @@
+# typed: false
+# frozen_string_literal: true
+
+class Pyroscope < Formula
+  desc "Open source continuous profiling software"
+  homepage "https://pyroscope.io"
+  url "https://dl.pyroscope.io/release/pyroscope-0.5.1-source.tar.gz"
+  sha256 "a5f138a04a7a3b31c9e693370d9c214e3a6724fe14fbe123f8b7adc4ae28aff3"
+  license "Apache-2.0"
+  head "https://github.com/pyroscope-io/pyroscope.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+  depends_on "rust" => :build
+  depends_on "yarn" => :build
+  depends_on "zstd" => :build
+
+  on_linux do
+    depends_on "libunwind"
+    depends_on "php"
+    depends_on "gcc"
```

Comment:
Does it not use these on macOS?

Reply:
it doesn't, only on Linux.

Reply:
Why would it only use php on Linux?

Reply:
test

Reply:
Because the dependency `phpspy` is only available to linux.